taxi/forms.py

```diff
@@ -1,3 +1,5 @@
+from logging import fatal
```

Comment:
The import statement `from logging import fatal` is incorrect. `fatal` is not a module or class that can be imported directly from `logging`. If you need to use logging, consider importing the `logging` module itself and using `logging.fatal()`.